src/jit/gentree.h

```diff
@@ -4369,20 +4435,20 @@ struct GenTreePutArgStk : public GenTreeUnOp
 
     GenTreePutArgStk(genTreeOps oper,
                      var_types  type,
-                     unsigned slotNum FEATURE_UNIX_AMD64_STRUCT_PASSING_ONLY_ARG(unsigned numSlots)
-                         FEATURE_UNIX_AMD64_STRUCT_PASSING_ONLY_ARG(bool isStruct),
+                     unsigned slotNum PUT_STRUCT_ARG_STK_ONLY_ARG(unsigned numSlots)
+                         PUT_STRUCT_ARG_STK_ONLY_ARG(bool isStruct),
```

Comment:
I will leave this as-is for the moment. I think there is other cleanup required here in any case.

---
In reply to: [79513258](https://github.com/dotnet/coreclr/pull/7252#discussion_r79513258) [](ancestors = 79513258)